distribution/sender/routes/sender-activemq-publisher.xml

```diff
@@ -0,0 +1,149 @@
+                <log message="Fetched ${body.size()} sender sync messages" />
+
+                <split parallelProcessing="false">
+                    <simple>${body}</simple>
+                    <setProperty name="senderSyncMessage">
+                        <simple>${body}</simple>
+                    </setProperty>
+
+                    <log message="Preparing sender sync message -> ${exchangeProperty.senderSyncMessage}" />
+                    
+                    <choice>
+                        <when>
+                            <simple>${exchangeProperty.senderSyncMessage.operation} == 'd'</simple>
+                            <setBody>
+                                <spel>#{new org.openmrs.eip.component.model.SyncModel()}</spel>
+                            </setBody>
+                            <script>
+                                <spel>
+                                    #{body.setTableToSyncModelClass(T(org.openmrs.eip.component.service.TableToSyncEnum).getTableToSyncEnum(getProperty('senderSyncMessage').tableName).modelClass)}
+                                    #{body.setModel(body.tableToSyncModelClass.getConstructor().newInstance())}
+                                    #{body.model.setUuid(getProperty('senderSyncMessage').identifier)}
+                                    #{body.setMetadata(new org.openmrs.eip.component.model.SyncMetadata())}
+                                </spel>
+                            </script>
+            
+                            <log message="Deleted entity payload -> ${body}" />
+                        </when>
+                        <otherwise>
+                            <log message="Loading entity from DB..." loggingLevel="DEBUG" />
+            
+                            <toD uri="openmrs:extract?tableToSync=${exchangeProperty.senderSyncMessage.tableName.toUpperCase()}&amp;uuid=${exchangeProperty.senderSyncMessage.identifier}" />
+            
+                            <log loggingLevel="DEBUG" message="Loaded entity -> ${body}" />
+            
+                            <choice>
+                                <when>
+                                    <simple>${body.size()} == 1</simple>
+                                    <setBody>
+                                        <jsonpath>$[0]</jsonpath>
+                                    </setBody>
+                                </when>
+                                <otherwise>
+                                    <setBody>
+                                        <simple>${null}</simple>
+                                    </setBody>
+                                </otherwise>
+                            </choice>
+                        </otherwise>
+                    </choice>
+            
+                    <choice>
+                        <when>
+                            <simple>${body} != null || ${exchangeProperty.event.operation} == 'r'</simple>
```

Comment:
This condition looks incorrect, there is no longer an exchange property named event, so this logic is broken